util.py

```diff
@@ -0,0 +1,362 @@
+from __future__ import annotations
+from configparser import ConfigParser
+from contextlib import AbstractContextManager, nullcontext
+from copy import deepcopy
+from enum import Enum, auto
+import fcntl
+import functools
+from io import TextIOWrapper
+import logging
+from pathlib import Path
+from threading import Lock, RLock, Timer, Condition
+from typing import Callable, TypedDict
+import pigpio
+import influxdb_client
+from influxdb_client.client.write_api import SYNCHRONOUS
+from influxdb_client.rest import ApiException
+from urllib3.exceptions import NewConnectionError
+
+
+CONFIG = {
+    "config_file": "/etc/mini-air-quality/sensors_config.ini",
```

Comment:
`sensors_config_file` to align convention? And follow?
Please consider.

Reply:
Changed to sensor_file and same for others so I don't duplicate names: i.e. `CONFIG["config..."]`